### PR TITLE
Add "Unix Timestamp" extension

### PR DESCRIPTION
**Pull request by Destiner**
## Description

Adds a new extension. The extension allows to convert unix timestamp (e.g. `1639395980`) to a human readable date string, and vice versa. Also, it allows to copy the current timestamp to the clipboard.

## Type of change

- New extension

## Screencast

https://user-images.githubusercontent.com/4247901/145808590-3885553b-d835-40ee-9136-ccb2ff8d0c4b.mov

## Notes

* I wasn't sure about extension's name. There are various ways to name the same concept: timestamp, unix timestamp, unix time. I picked the one that's I'm most familiar with.
* I tried to keep the extension's UI simple via using `<Toast>`s extensively.
* I decided not to use `<Form.DatePicker>` for date input. It fits greatly for more natural user input ("2 days ago", "next Monday"), but when dealing with timestamps, you usually have a very specific date/time in mind. Also, AFAIK it doesn't support seconds.

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
Hi @Destiner 🔥 

Are you aware of that there is a toolbox extension having almost identically features (#472) – Could it be an idea to contribute to that extensions instead?

**Comment by Destiner** [comment]
Hey @pernielsentikaer, thanks for the feedback!

I wasn't aware of Toolbox. It indeed has some related functionality, and I could contribute to it. However, it seems that it uses a different way to process input. Also, I'm not sure how the "get current timestamp" command would fit, since Toolbox seems to focus on formatting only.

I was looking for an extension named similarly in the store ('timestamp', 'time'), and couldn't find any so I assumed there isn't something like this. I'd expect many others would do a similar query.

So I'm not sure whether it's a good idea. But it's your call, and if you think it doesn't suffice for a separate extension, I can talk to Toolbox's author about collaboration.

P.S. I was also thinking about creating a more generic extension covering more time-related utilities (timezone differences, parse date string). Is that a good idea?

**Comment by kkange** [comment]
Hi @pernielsentikaer @Destiner 

The ToolBox extension certainly has that feature.  
However, in order to integrate all other functions, individual functions are implemented to a minimum.  
Currently, we do not plan to provide a customized environment for each function.  
I think it is better to release it as a different extension to provide more various settings and functions.

**Comment by mattisssa** [comment]
Hey @Destiner! Speaking of timezone differences and similar things, are you aware that all this is already possible with [Calculator](https://www.raycast.com/extensions/calculator/)?
CleanShot 2021-12-16 at 15 26 04

**Comment by Destiner** [comment]
Hey @mattisssa. Yeah, I know about this feature and it's great. I was thinking about something like showing a list of timezones. Anyway, it's just one idea.

**Comment by Friss** [comment]
It would be awesome for either this extension or toolbox to land this feature. Last night I was close to building this myself as well because of how useful it is.

**Comment by pernielsentikaer** [comment]
Hi @Destiner 

So is it about timezones or Unix timestamp? I think if you want to go more through the timezone, then should the extension and name match this 💯

**Comment by Destiner** [comment]
Hey @pernielsentikaer 

The extension is about unix timestamp utils. Do you think anything's missing in the extension before merging it?

File: extensions/unix-timestamp/src/timestamp-to-date.tsx
Comment (fe9lix) [inline extensions/unix-timestamp/src/timestamp-to-date.tsx:26]: Looks like a `return` statement is missing here?

File: extensions/unix-timestamp/src/timestamp-to-date.tsx
Comment (fe9lix) [inline extensions/unix-timestamp/src/timestamp-to-date.tsx:30]: Same here? (missing `return`)

File: extensions/unix-timestamp/src/timestamp-to-date.tsx
Comment (Destiner) [reply extensions/unix-timestamp/src/timestamp-to-date.tsx:26]: Right, fixed

File: extensions/unix-timestamp/src/timestamp-to-date.tsx
Comment (Destiner) [reply extensions/unix-timestamp/src/timestamp-to-date.tsx:31]: Fixed as well

**Review by fe9lix — APPROVED** [review]

**Pull request merged**
